src/Storage/Doctrine/PhpcrOdm/ContentTypeDriver.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Symfony\Cmf\Component\ContentType\Storage\Doctrine\PhpcrOdm;
+
+use Doctrine\Common\Persistence\Mapping\Driver\MappingDriver;
+use Symfony\Cmf\Component\ContentType\MappingBuilderCompound;
+use Symfony\Cmf\Component\ContentType\MappingInterface;
+use Symfony\Cmf\Component\ContentType\FieldRegistry;
+use Symfony\Cmf\Component\ContentType\Mapping\StringMapping;
+
+class ContentTypeDriver implements MappingDriver
```

Comment:
This is an initial draft of a metadata driver for the PHPCR-ODM. Currently it maps field models (e.g. the `Image` DTO of an image field). We could also map the scalar fields of the main entity, saving the user from duplicating information (i.e. mapping both `@CmfContentType\Field('text')` and `@ODM\Field(type='string')` on the same property).